.github/workflows/ci.yml

```diff
@@ -69,6 +69,33 @@ jobs:
       - name: Test async-std-runtime
         run: cargo test --features async-std-runtime
 
+  autobahn:
+    name: Autobahn tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+          - nightly
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          profile: minimal
+          override: true
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install python-unittest2
+          sudo pip install urllib3[secure] autobahntestsuite
```

Comment:
These don't have to be installed anymore for the other tests anymore then, those then only need `libssl-dev` :)

Reply:
`libssl-dev` is not listed as [installed] is already part of the base image but i included it anyway

[installed]: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md